### PR TITLE
feature/113: created content app

### PR DESCRIPTION
**Pull request by kleikoks**

### CODE REVIEW

**Review by JedrzejMal — APPROVED** [review]

File: src/apps/content/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {% block header %}
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block meta %}
+    <title>Awesome ecommerce</title>
+    {% endblock meta %}
+
+    {% endblock header %}
```
Comment (egorvavilov) [inline src/apps/content/templates/base.html]: header is body block

**Review by egorvavilov — CHANGES_REQUESTED** [review]

File: src/apps/content/templates/base.html (hunk 1 above)
Comment (egorvavilov) [reply src/apps/content/templates/base.html]: `<head>` html block is a home for seo meta tags

`<body>` is home for header/footer/content and other valuable blocks

Comment (kleikoks) [reply src/apps/content/templates/base.html]: My bad. Moved it to the body block.

**Review by egorvavilov — APPROVED** [review]
LGTM

**Pull request merged**